scripts/cbmc_json_parser.py

```diff
@@ -72,6 +76,42 @@ def main():
     return_code = transform_cbmc_output(sample_json_file_parsing, output_style=output_style)
     sys.exit(return_code)
 
+
+class SourceLocation:
+    def __init__(self, source_location={}):
+        """ Coverts a source location dictionary from CBMC json into an object.
+           The SOURCE_LOCATION_OBJECT has the following structure:
+           {
+             'column': '<num>',
+             'file': '<file_name>',
+             'function': '<fn_name>',
+             'line': '<num>'
+           }},
+
+           Some fields might be missing.
+        """
+        self.fname = source_location.get("file", None)
```

Comment:
nit: fname isn't great, I thought it was function name until I saw the next line. Just use filename?

Reply:
For sure!